### PR TITLE
java-service-wrapper 3.5.56

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

<details>
<summary>license ref</summary>

```
                         Tanuki Software, Ltd. 
                 Community Software License Agreement
                             Version 1.3

IMPORTANT-READ CAREFULLY: This license agreement is a legal agreement
between you ("Licensee") and Tanuki Software, Ltd. ("TSI"), which
includes computer software, associated media, printed materials, and
may include online or electronic documentation ( Software ).  PLEASE
READ THIS AGREEMENT CAREFULLY BEFORE YOU INSTALL, COPY, DOWNLOAD OR
USE THE SOFTWARE ACCOMPANYING THIS PACKAGE.

Section 1 - Grant of License

Community editions of the Software are made available on the GNU
General Public License, Version 2 ("GPLv2") or Version 3 ("GPLv3"),
included in Sections 4 and 5 of this license document.  All sections
of the Community Software License Agreement must be complied with in
addition to those of either the GPLv2 or GPLv3.  This license allows
the Software Program to be used with Products that are released under
either GPLv2 or GPLv3.


Section 2 - Definitions 

2.1. "Community Edition" shall mean versions of the Software Program
distributed in source form under this license agreement, and all new
releases, corrections, enhancements and updates to the Software
Program, which TSI makes generally available under this agreement.

2.2. "Documentation" shall mean the contents of the website
describing the functionality and use of the Software Program, located
at http://wrapper.tanukisoftware.org

2.3. "Product" shall mean the computer programs, that are provided by
Licensee to Licensee customers or potential customers, and that
contain both the Software Program as a component of the Product, and a
component or components (other than the Software Program) that provide
the material functionality of the Product.  If the Product is released
in source form, the Software Program or any of its components may only
be included in executable form.

2.4. "Software Program" shall mean the computer software and license
file provided by TSI under this Agreement, including all new releases,
corrections, enhancements and updates to such computer software, which
TSI makes generally available and which Licensee receive pursuant to
Licensee subscription to TSIMS. Some specific features or platforms
may not be enabled if they do not fall under the feature set(s)
covered by the specific license fees paid.

2.5 "End User" shall mean the customers of the Licensee or any
recipient of the Product whether or not any payment is made to use
the Product.


Section 3 - Licensee Obligations

A copy of this license must be distributed in full with the Product
in a location that is obvious to any End User.

In accordance with Section 4 (GPLv2) or Section 5 (GPLv3), the full
source code of all components of the Product must be made available to
any and all End Users.

Licensee may extend and/or modify the Software Program and distribute
under the terms of this agreement provided that the copyright notice
and license information displayed in the console and log files are
not obfuscated or obstructed in any way.
```

</details>

### CODE REVIEW

**Comment by chenrui333** [comment]
> Should be able to use latest OpenJDK from what I've seen.
> 
> EDIT: See last line of the block here: https://wrapper.tanukisoftware.com/doc/english/introduction.html#SupportedJVM

does not build with latest jdk

File: .github/autobump.txt [hunk 1]
```diff
@@ -842,7 +842,6 @@ ivykis
 jackett
 jadx
 jaq
-java-service-wrapper
```
Comment (SMillerDev) [inline .github/autobump.txt:845]: Why?

Comment (chenrui333) [reply .github/autobump.txt:845]: it cannot be bumped, https://github.com/Homebrew/homebrew-core/actions/runs/8133100326/job/22224266910#step:6:8443

```
==> java-service-wrapper
Current formula version:  3.5.54
Latest livecheck version: 3.5.56
Latest Repology version:  3.5.54
Open pull requests:       none
Closed pull requests:     none
Error: These open pull requests may be duplicates:
java-service-wrapper 3.5.56 https://github.com/Homebrew/homebrew-core/pull/164916
Duplicate PRs should not be opened.
Manually open these PRs if you are sure that they are not duplicates.
```

Comment (chenrui333) [reply .github/autobump.txt:845]: there is a dynamic date value in the download url.

Comment (SMillerDev) [reply .github/autobump.txt:845]: Shouldn't we fix that instead? That seems like a livecheck issue.

Comment (SMillerDev) [reply .github/autobump.txt:845]: @samford  any suggestions?

Comment (chenrui333) [reply .github/autobump.txt:845]: as you can see the livecheck actually works fine, it is more of the bump issue

**Comment by chenrui333** [comment]
rebase to resolve the merge conflict

**Review by MikeMcQuaid — APPROVED** [review]

**Pull request merged**